docs/PUBLISHING.md

```diff
@@ -27,10 +27,9 @@ We follow a [git-flow](https://nvie.com/posts/a-successful-git-branching-model/)
 
 #### Cutting a release branch
 
-_Warning: In step 4 you will see a message in your terminal with instructions on publising the package. Ignore the message in your terminal and follow the instructions in this guide instead. We send the release branch through the pull request review process before publishing._
-
+0. Before beginning, run `git checkout next && git pull origin next` to ensure you have the most up-to-date changes.
 1. Confirm that all checks are green on CI.
-2. Run `git checkout next && git pull origin next` to ensure you have the most up-to-date changes.
+2. Run `git fetch origin` to retrieve all the latest remote branches and tags.
```

Comment:
Was trying to think of a way to join steps 0 and 2 but this seems safer

Reply:
OH, i switched these two around lol. 0 should be to fetch origin, and 2 should remain the same